acs/acs.go

```diff
@@ -0,0 +1,37 @@
+package acs
+
+type ACS interface {
+	Receive(batch []byte)
+	sendConsensus()
+	sendBatch(id int)
+	sendBinary(id int)
+}
+
+type memACS struct {
+	// TODO : ACS must have rbc instances map
```

Comment:
I think We need several channels to communicate with `RBC` and `BBA`.

Reply:
so I write rbc instances map.